src/lib/components/pages/TicketsOverview.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import { Filter, TicketCard } from '$lib/index'
+  export let itemCollection
+  export let cities
+</script>
+
+<section class="body">
```

Comment:
Deze classes zin overbodig, het is sveltekit! Gebruik betere css selectoren...

Reply:
Ik heb de classes weg gehaald in [de meest recente commit](https://github.com/fdnd-agency/wogo/pull/24/commits/2ff0eece8f24182bac71bc0f3e425676e9de5220). Ik maak nu gebruik van de css selectors ":first-child" & ":last-child" en ik maak gebruik van een child combinator om een section in een section te targeten